src/index.js

```diff
@@ -1,18 +1,21 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
 // import {BrowserRouter as Router} from "react-router-dom"
-import { BrowserRouter } from 'react-router-dom'
-import { AuthProvider } from './Hooks/Context/authContext';
+import { BrowserRouter } from "react-router-dom";
+import { AuthProvider } from "./Hooks/Context/authContext";
```

Comment:
try to use one index.js file to explore all global states from one place

Reply:
okey I will keep in mind from next time